tests/php/Wordpress/CustomPostTypeTest.php

```diff
@@ -420,21 +429,42 @@ protected function createLocation($title, $postStatus = 'publish', $admins = [],
 
 		$this->locationIds[] = $locationId;
 
-		if (! empty($admins)) {
+		if ( ! empty( $admins ) ) {
 			update_post_meta( $locationId, COMMONSBOOKING_METABOX_PREFIX . 'location_admins', $admins );
 		}
 
 		return $locationId;
 	}
 
-	protected function createMap($options) {
+	protected function createMap( $options = null ) {
 		$mapId = wp_insert_post( [
 			'post_title'  => 'Map',
 			'post_type'   => Map::$postType,
 			'post_status' => 'publish'
 		] );
 
-		update_post_meta( $mapId, 'cb_map_options', $options );
+		//TODO: This is the old map format, remove old tests that are dependent on it
```

Comment:
@hansmorb hier ist noch ein TODO.

Reply:
Stimmt! Dann müssten wir nch testen, ob createMap auch wirklich immer mit entsprechenden Optionen aufgerufen wird.

Reply:
Ich finde nur einen Aufruf in CustomPostType/CustomPostTypeTest:293 ... sieht aus als wäre der Test durch diesen PR nicht angefasst worden.